packages/lesswrong/components/seasonal/petrovDay/petrov-day-story/PetrovDayStory.tsx

```diff
@@ -0,0 +1,393 @@
+"use client";
+import CloudinaryImage2 from '@/components/common/CloudinaryImage2';
+import { defineStyles } from '@/components/hooks/defineStyles';
+import { useStyles } from '@/components/hooks/useStyles';
+import { AnalyticsContext } from '@/lib/analyticsEvents';
+import React from 'react';
+import { petrovDaySections } from './petrovDaySectionsFinal';
+import ContentStyles from '@/components/common/ContentStyles';
+import classNames from 'classnames';
+
+const styles = defineStyles("PetrovDayStory", (theme: ThemeType) => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    // Custom scroll cursor with hotspot at (10,10) and pointer fallback
+    cursor: 'url("/icons/scroll.png") 10 10, pointer',
```

Comment:
if you do put the images into cloudinary and delete them locally, this might not work anymore (idk) so might want to skip doing this one